src/main/webapp/app/exercises/shared/exercise-headers/header-exercise-page-with-details.component.html

```diff
@@ -21,6 +21,13 @@ <h4 *ngIf="!isTestRun">
                 </h4>
             </ng-template>
             <ng-template #courseMode>
+                <h4 *ngIf=exercise "notReleased()">
```

Comment:
I fixed the *ngIf, but your suggestion does unfortunately not work because I cannot use moment in the html file.

Reply:
Can you do it like exerciseStatusBadge so that you don't use function inside HTML? Also there must be at least quotes around exercise :D *ngIf="exercise"